wgsl/index.bs

```diff
@@ -13830,6 +13830,7 @@ but a value may infer the type.
     * When `e` is zero, the fraction is zero.
     * When `e` is non-zero and normal, `e` &equals; `fraction * 2`<sup>`exponent`</sup>, where
         the fraction is in the range [0.5, 1.0) or (-1.0, -0.5].
+    * When `e` is denormalized, the fraction and exponent are [=indeterminate values=].
```

Comment:
This is the simplest and least strict option.

ECMAScript hints that denormalized numbers are not supported.

https://262.ecma-international.org/13.0/#sec-number.min_value says (emphasis added)
> In the [IEEE 754-2019](https://262.ecma-international.org/13.0/#sec-bibliography) double precision binary representation, the smallest possible value is a denormalized number. **If an implementation does not support denormalized values,** the value of Number.MIN_VALUE must be the smallest non-zero positive value that can actually be represented by the implementation.